### ISSUE TITLE
Cannot build natively on Arch Linux

### ISSUE THREAD

**Issue by grakshith** [pixelspark/catena#134]
![screenshot from 2018-08-10 20-54-22](https://user-images.githubusercontent.com/10547321/43966482-a7bfd4e4-9cdf-11e8-85a8-bedd5a33bfd5.png)

System:
Arch linux
Swift - 4.1
All required libraries present.

**Comment by pixelspark** [comment]
Looks like it is unable to find the OpenSSL library. Did you install the openssl library+headers? (See the Dockerfile, you should at least execute the equivalent of `apt-get install libcurl4-openssl-dev` on Arch)

**Comment by grakshith** [comment]
Yes, I have installed all the equivalent packages of the Debian ones on Arch. However, I am still unable to build it.

**Comment by lf-araujo** [comment]
Hi @hehaichi , the problem is with BlueSSL, which haven't quite yet released a version with support for latest SSL. Unfortunately we either wait for the author, or we add a PR. 

Also note that with latest Arch updates and glibc changes, swift will no longer build in Arch.
